introduction.md

```diff
@@ -0,0 +1 @@
+Hello I am Akanchha Singh Bhadauria.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
My favorite Code School path is HTML

Reply:
My favorite Code School path is HTML